### ISSUE TITLE
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.io.File.getAbsolutePath()' on a null object reference

### ISSUE THREAD

**Issue by waqarmustafa3** [akshay2211/PixImagePicker#162]
on Android 11 it's not working please check it out

**Comment by iadcialim** [comment]
Same issue here
```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.io.File.getAbsolutePath()' on a null object reference
        at com.fxn.pix.Pix$8$1.onFileReady(Pix.java:457)
        at com.otaliastudios.cameraview.CameraUtils$1$1.run(CameraUtils.java:124)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by akshay2211** [comment]
Hello @waqarmustafa3 
This issue is been resolved in the latest [PixImagePicker version 1.5.5](https://github.com/akshay2211/PixImagePicker#download)
